content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -52,6 +52,16 @@ The following apply to every installation:
 - `capacity_concurrency` — number of concurrent plans and applies; defaults to `10`
 - `extra_no_proxy` — (Optional) when configured to use a proxy, a `,` (comma) separated list of hosts to exclude from proxying
 - `ca_certs` — (Optional) custom certificate authority (CA) bundle
+- `cust_vault_enable` - (Optional) Indicate if an external Vault cluster is being used. Set to `"1"` if so.
```

Comment:
Other numbers in this doc are referenced without the double quotes.  The settings file does require strings for all values, but that's mentioned elsewhere. For consistency, let's to just `1` here.